satpy/readers/netcdf_utils.py

```diff
@@ -187,6 +191,12 @@ def collect_cache_vars(self, cache_vars, obj):
 
     def __getitem__(self, key):
         """Get item for given key."""
+        if key in ('/attr', '/attrs'):
```

Comment:
As you can see in the checks, Codebeat is complaining that this method has become too nested: https://codebeat.co/projects/github-com-pytroll-satpy-main/pull_requests/487392

If you are up to it, you could try refactoring a bit, maybe to something like this:
```python
def __getitem__(self, key):
    """Get item for given key."""
    if key in ('/attr', '/attrs'):
        return self._get_global_attrs()
    val = self.file_content[key]
    if isinstance(val, netCDF4.Variable):
        return self._get_variable(key, val)
    elif isinstance(val, netCDF4.Group):
        return self._get_group(key, val)
```
and create the new methods `_get_global_attrs`, `_get_var`, `_get_group` by moving the corresponding code in there.

Reply:
Thanks for the comment. I've moved those!